src/geometry/ArborX_HyperPoint.hpp

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_HYPERPOINT_HPP
+#define ARBORX_HYPERPOINT_HPP
+
+// FIXME_CXX17: remove first clause and update the comment once we require C++17
+#if (__cplusplus < 201703L) || (KOKKOS_VERSION < 30699)
+#error "Using multi-dimensional Point requires C++17 and Kokkos 3.7"
+#endif
+
+#include <ArborX_GeometryTraits.hpp>
+
+#include <Kokkos_Macros.hpp>
+
+#include <utility>
```

Comment:
What do we need that header include for?

Reply:
Leftover from some experimentation, removed.